service/builder.go

```diff
@@ -22,22 +22,26 @@ package service
 
 import "github.com/pkg/errors"
 
-// WithModules is a helper to create a service without any options
-func WithModules(modules ...ModuleCreateFunc) *Builder {
```

Comment:
You can't do this anymore, `WithModule` takes more than one parameter for every module.